pipe/cent.py

```diff
@@ -34,7 +34,7 @@ def psf(psf_spline, frame, noise, xc, yc,
     """Use a PSF to find the best matching centre in a frame.
     Uses noise frame to clip bad pixels.
     """
-
+    from .reduce import coo_mat
```

Comment:
Reverting [this change](https://github.com/alphapsa/PIPE/commit/47fefcaa1b06199b2d0196f610696854758067ab#r48178624).